Assets/Scripts/PathMovement.cs

```diff
@@ -11,12 +11,16 @@ public class PathMovement : MonoBehaviour {
     public GameObject pathPrefab;
     private GameObject instantiatedPath;
     public float speed = 1.0f;
+    public float attackDelay;

```

Comment:
Ideally we would put this to its own component, because path movement and attack are two different behaviors. But since @Xavier-Gibbon added this behavior to path (too late to change now), making those changes are fine here.